#goal definition
std_msgs/String ask_response
---
#result definition
---
#feedback